shared/utils.py

```
@@ -109,3 +109,43 @@ def wrapper(*args, **kwargs):
     return wrapper
+
+
+def iso8601_to_unix_ms(iso_timestamp) -> int:
+    """ISO 8601タイムスタンプまたはdatetimeオブジェクトをUnixミリ秒に変換します。
+
+    Args:
+        iso_timestamp: ISO 8601形式のタイムスタンプ文字列、またはdatetimeオブジェクト
+                      (例: "2025-12-14T02:30:00.000Z" または datetime(2025, 12, 14, 2, 30))
+
+    Returns:
+        Unixエポックからのミリ秒（整数）
```

Ruff (0.14.8)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)